spi/src/main/java/org/opensearch/jobscheduler/spi/utils/LockService.java

```diff
@@ -44,7 +45,7 @@
 
 public final class LockService {
     private static final Logger logger = LogManager.getLogger(LockService.class);
-    private static final String LOCK_INDEX_NAME = ".opendistro-job-scheduler-lock";
```

Comment:
While we are making a change of this nature can we rename the index to opensearch?

Reply:
Resolved this on the security plugin change to add the system index - since the the index name isn't change at all, it would introduce much more risk for little gain.